tools/project.py

```diff
@@ -128,77 +125,68 @@
         if exists(EXPORT_DIR):
             rmtree(EXPORT_DIR)
 
+
+    # Target
+    if options.mcu is None :
```

Comment:
These 2 (mcu, ide) are not required - reason: https://github.com/mbedmicro/mbed/pull/2141/files